library/coretests/tests/num/flt2dec/random.rs

```diff
@@ -203,9 +203,6 @@ fn shortest_f64_hard_random_equivalence_test() {
 }
 
 #[test]
-<<<<<<< HEAD
```

Comment:
I am not sure this conflict was correctly resolved? This attribute moved to a new test?

Reply:
GOt it! Thanks!